supertinycron/README.md

```diff
@@ -0,0 +1,15 @@
+## Cron expression parsing in ANSI C
+
+Given a cron expression and a date, you can get the next date which satisfies the cron expression. The more details please refer to [README.md](https://github.com/espressif/idf-extra-components/blob/master/supertinycron/supertinycron/README.md)
+
+## Usage Example
+
+Refer to [cron_example](https://github.com/espressif/idf-extra-components/blob/master/supertinycron/examples/cron_example)
```

Comment:
I've looked into the example, but quite honestly it looks more like a unit test for the library. It is probably a really good way to check if the library works, but perhaps we can show something simpler as an example?

If you don't have time for this, I'm okay merging the current version.

Reply:
Yes, this is actually a unit test. There is an introduction to the simple use of code in the README. For the current version, let’s use unit test as an example first.